dli_app/mod_auth/controllers.py

```diff
@@ -103,3 +115,67 @@ def logout():
         logout_user()
     flash('You have successfully logged out.', 'alert-success')
     return redirect(url_for('default.home'))
+
+@mod_auth.route('/resetpass/', methods=['GET', 'POST'])
+def resetpass():
+    """Reset the user's password
+
+    If the user successfully submitted the form, send a password 
+    reset email. Otherwise, render the reset form again.
+    """
+
+    form = ForgotForm()
+    if form.validate_on_submit():
+        # Email is authenticated and sent
+        email = form.email.data
+	#Create User Specific URL here
+        pw_reset = PasswordReset(
+            user= User.get_by_email(email),
+            key=''.join(
+                random.choice(
+                    string.ascii_letters + string.digits
+                ) for _ in range(60)
+            ),
+        )
+        db.session.add(pw_reset)
+        db.session.commit()
+        #ADD EMAIL STUFF HERE
+	mail=Mail(current_app)
+	title='Reset your Password'
+        url="http://68.234.146.84:PORT /auth/setnewpass/"+pw_reset.key
+        content='Click this link to reset your password: '+url
+	sender='cs490testing@gmail.com'
+	msg=Message(title,sender=sender,recipients=[email])
+	msg.body=content
+	mail.send(msg)
```

Comment:
This section has a lot of inconsistent spacing.
Also, only use var=value in function calls. In general, you should put a single space around operators
```python
mail = Mail(current_app)
title = 'Reset your Password'
etc.
```